examples/kotlin/WORKSPACE

```diff
@@ -0,0 +1,74 @@
+workspace(
+    name = "kotlin_example",
+    managed_directories = {"@npm": ["node_modules"]},
+)
+
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+http_archive(
+    name = "build_bazel_rules_nodejs",
+    sha256 = "da72ea53fa1cb8ab5ef7781ba06b97259b7d579a431ce480476266bc81bdf21d",
+    urls = ["https://github.com/bazelbuild/rules_nodejs/releases/download/0.36.2/rules_nodejs-0.36.2.tar.gz"],
+)
+
+# Install external npm dependencies
+load("@build_bazel_rules_nodejs//:defs.bzl", "npm_install")
+
+npm_install(
+    # Name this npm so that Bazel Label references look like @npm//package
+    name = "npm",
+    package_json = "//:package.json",
+    package_lock_json = "//:package-lock.json",
+)
+
+# Install any Bazel rules which were extracted earlier by the yarn_install rule.
+load("@npm//:install_bazel_dependencies.bzl", "install_bazel_dependencies")
+
+install_bazel_dependencies()
+
+# Install external Kotlin/Java dependencies
+http_archive(
+    name = "rules_jvm_external",
+    sha256 = "f04b1466a00a2845106801e0c5cec96841f49ea4e7d1df88dc8e4bf31523df74",
+    strip_prefix = "rules_jvm_external-2.7",
+    url = "https://github.com/bazelbuild/rules_jvm_external/archive/2.7.zip",
+)
+
+load("@rules_jvm_external//:defs.bzl", "maven_install")
+
+maven_install(
+    name = "maven",
+    artifacts = [
```

Comment:
Consider adding a comment with instructions to update maven_install.json

```
# Run 'bazel run @unpinned_maven//:pin` after updating this list to create a new maven_install.json
```

Running this command is not very intuitive, and can save people a few minutes from reading the docs.